cmd/ferretdb/main.go

```diff
@@ -124,6 +125,8 @@ var cli struct {
 			ReportTimeout  time.Duration `default:"5s"                           help:"Telemetry: report timeout."`
 			Package        string        `default:""                             help:"Telemetry: custom package type."`
 		} `embed:"" prefix:"telemetry-"`
+
+		OTLPEndpoint string `default:"" help:"OTLP exporter endpoint, if unset, OpenTelemetry is disabled."`
```

Comment:
Please move it before telemetry to other experimental flags (and "Experimental: " should be in the help text)